syntax = "proto3";

message Message {
  // Define fields for your message
  int32 fnccode = 1;
  string param1 = 2;
  string param2 = 3;
  string param3 = 4;
  // Add more fields as needed
}